runtime/src/serde_snapshot/newer.rs

```diff
@@ -425,13 +446,51 @@ impl<'a> TypeContext<'a> for Context {
             epoch_accounts_hash.copied(),
             matches!(epoch_reward_status, EpochRewardStatus::Active(_))
                 .then_some(&epoch_reward_status),
+            use_new_epoch_stakes_field,
         ) {
             BankFieldsToSerialize::WithoutEpochRewardStatus(data) => {
                 bincode::serialize_into(stream_writer, &data)
             }
             BankFieldsToSerialize::WithEpochRewardStatus(data) => {
                 bincode::serialize_into(stream_writer, &data)
             }
+            BankFieldsToSerialize::WithVersionedEpochStakes(data) => {
+                bincode::serialize_into(stream_writer, &data)
+            }
+        }
+    }
+}
+
+#[derive(AbiExample, AbiEnumVisitor, Debug, Clone, PartialEq, Serialize, Deserialize, Default)]
+pub(crate) enum UnusedEnum {
+    #[default]
+    Unused = 1,
+}
```

Comment:
This represents the unused epoch rewards status field

Reply:
I don't grok why this is needed. I get that it matches `EpochRewardStatus::Inactive`, but we would only serialize that field in a snapshot if the partitioned-epoch rewards feature were active and the snapshot were made in the rewards period, ie `EpochRewardStatus::Active(_)`.

Reply:
Ah, I see. I think that's up for discussion.